### PR TITLE
Fix static link option

### PR DESCRIPTION
**Pull request by hlts2**
### Description:

I fixed static link option bug.

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kpango** [comment]
please add -installsuffix netgo

File: dockers/agent/ngt/Dockerfile [hunk 1]
```diff
@@ -60,6 +60,7 @@ RUN GO_VERSION="$(cat GO_VERSION)" \
     CGO_LDFLAGS="-g -Ofast -march=native" \
     GO111MODULE=on \
     go build \
+    -a -tags netgo \
```
Comment (rinx) [inline dockers/agent/ngt/Dockerfile]: badge
agent-ngt has already built with these options.
https://github.com/vdaas/vald/blob/aee9fd4b52acf62ceab5debc1b06c79037dbea99/dockers/agent/ngt/Dockerfile#L72-L73

File: dockers/discoverer/k8s/Dockerfile [hunk 2]
```diff
@@ -45,6 +45,7 @@ RUN GO_VERSION="$(cat GO_VERSION)" \
     && CGO_ENABLED=0 \
     && GO111MODULE=on \
     go build \
+    -a -tags netgo \
```
Comment (rinx) [inline dockers/discoverer/k8s/Dockerfile]: badge
could you format the order of the newly added options like agent-ngt?
https://github.com/vdaas/vald/blob/aee9fd4b52acf62ceab5debc1b06c79037dbea99/dockers/agent/ngt/Dockerfile#L72-L73

File: dockers/agent/ngt/Dockerfile (hunk 1 above)
Comment (hlts2) [reply dockers/agent/ngt/Dockerfile]: :bow:

**Pull request merged**